### PR TITLE
Add guide to CPG budgets

### PR DESCRIPTION
**Pull request by illusional**
I've drafted a few cases, but happy to take more suggestions for examples I can write out an action plan for.

### CODE REVIEW

File: budgets.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Storage:
+
+- Standard $0.023 USD per GB per month (~$38 AUD per TB per month)
+- Nearline $0.016 USD per GB per month (~$26 AUD per TB per month)
+- Coldline $0.006 USD per GB per month (~$10 AUD per TB per month)
+
+
+## Situations
+
+> I'm about to 2.5 TB of data into $DATASET
```
Comment (cassimons) [inline budgets.md:28]: ingest?

File: budgets.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+2. Bump the GCP budget for the project to $300 AUD per month, by submitting a pull request to [cpg-infrastructure-private `$DATASET/budgets.yaml`](https://github.com/populationgenomics/cpg-infrastructure-private) file.
+3. Reviewers will be automatically selected, and the budget will get updated shortly after merging.
+
+
+> I'm running a large compute job for seqr, eg: alignment, variant calling, joint-calling, etc.
+
+1. You should make sure the budget for DATASET's GCP project is set to cover the extra files you intend to produce.
+    1. Follow the steps above to bump the budget for DATASET's project.
+1. Estimate the cost of analysis (in AUD) and ensure:
+    1. The GCP budget for `hail-295901` has been updated to include the AUD cost of your analysis (including the cost of running Hail + other people's analysis). This must be done manually through the [budgets page](https://console.cloud.google.com/billing/01D012-20A6A2-CBD343/budgets?organizationId=648561325637).
```
Comment (cassimons) [inline budgets.md:41]:
```suggestion
    1. The GCP budget for `hail-295901` has been updated to include the AUD cost of your analysis (including the cost of running Hail + other people's analysis). This must be done manually through the [budgets page](https://console.cloud.google.com/billing/01D012-20A6A2-CBD343/budgets?organizationId=648561325637). Currently any of the data team leads have the permissions to alter this budget.
```

File: budgets.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+Budgets at the CPG are managed _mostly_ through GitHub, and there are a few places where budgets need to be managed.
+
+Some _you should knows_ from the start:
+
+- Compute through Hail Batch does NOT get assigned to the GCP project, it gets billed to the Hail GCP project.
+    - Compute through Dataproc and notebooks DO get assigned to the GCP project.
+- Hail maintains its own set of [_cumulative_ budgets](https://batch.hail.populationgenomics.org.au/billing_limits), in USD.
+- GCP budgets are _monthly_ and _reset_ on the 1st of each month (in Pacific Time, about 5 PM AEST).
+    - GCP budgets are managed in cpg-infrastructure-private (except Hail).
```
Comment (cassimons) [inline budgets.md:11]:
```suggestion
    - GCP budgets are managed in [cpg-infrastructure-private](https://github.com/populationgenomics/cpg-infrastructure-private) (except Hail).
```

**Review by cassimons — APPROVED** [review]
Looks great thanks.  Few suggestions you can take or leave.

**Review by katiedelange — APPROVED** [review]
Looks great to me.

**Pull request merged**